boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/maven/liberty/AbstractLibertyMojo.java

```diff
@@ -88,5 +100,72 @@ private void createDefaultRuntimeArtifactIfNeeded() {
     public void execute() throws MojoExecutionException {
         createDefaultRuntimeArtifactIfNeeded();
     }
+    /*
+     * Check for environment variable being set for server port. 
+     * if set update bootstrap.properties to add the server port
+     */
+    protected void setServerPort() {
```

Comment:
I can appreciate why you did it this way.   

This lets you override the port calculated in package during run/start/debug without affecting the port used by the packaged artifact.

However it doesn't allow the same -Dserver.port=8081 override during package.

I'm thinking a better approach would be to incorporate the -D CLI property into the property calculation used in `io.openliberty.boost.common.utils.SpringBootUtil.generateLibertyServerConfig`  and `getSpringBootServerProperties`, and to do this calculation including writing the whole thing out to boostrap.properties every time.  

Also, instead of having a Maven/Gradle-specific second place to adjust properties, let's pass the needed properties in from the build environment in a Maven/Gradle-specific way as needed, and write the properties in one place.

This is a new thing, so let's make sure to discuss with the team and hopefully get at least one review besides myself.